rpxdock/score/score_functions.py

```diff
@@ -83,7 +83,9 @@ def sasa_priority(pos1, pos2, lbub, lbub1, lbub2, ressc1, ressc2, **kw):
          mscore = (side1 + side2) / 2
       
          #ncont_score
-         ncont_score = ( a / prob_max )*( 1  / (c * np.sqrt(2 * np.pi) * (ub - lb)) ) * np.exp( -(np.log(ub - lb) - b)**2 / (2*c**2) )
+         #ncont = ub - lb
+         ncont_unique = len(np.unique(pairs[lb:ub][:,0])) + len(np.unique(pairs[lb:ub][:,1]))
+         ncont_score = ( a / prob_max )*( 1  / (c * np.sqrt(2 * np.pi) * (ncont_unique)) ) * np.exp( -(np.log(ncont_unique) - b)**2 / (2*c**2) )
```

Comment:
is the new definition only for sasa priority function? there should be documentation somewhere with this differences I think?

Reply:
I'm not sure I understand. This change improves the accuracy of the sasa priority score function. The old definition should be deprecated IMO. I can provide documentation in the wiki.

Reply:
you answered :) it's only relevant to sasa priority. I would add documentation in how ncontact is defined very differently here then in the other functions.